src/components/views/donate/DonateIndex.tsx

```diff
@@ -40,7 +40,7 @@ import { DonatePageProjectDescription } from './DonatePageProjectDescription';
 const DonateIndex: FC = () => {
 	const { formatMessage } = useIntl();
 	const { isMobile } = useDetectDevice();
-	const { project, isSuccessDonation, hasActiveQFRound } = useDonateData();
+	const { project, successDonation, hasActiveQFRound } = useDonateData();
```

Comment:
why was this renamed?

Reply:
@jainkrati, it was renamed because the previous name, 'isSuccessDonation,' was incorrect as it suggested a boolean type, but it actually includes the success donation data.